src/urllib3/contrib/pyopenssl.py

```diff
@@ -425,9 +425,12 @@ def load_verify_locations(self, cafile=None, capath=None, cadata=None):
             cafile = cafile.encode('utf-8')
         if capath is not None:
             capath = capath.encode('utf-8')
-        self._ctx.load_verify_locations(cafile, capath)
-        if cadata is not None:
-            self._ctx.load_verify_locations(BytesIO(cadata))
+        try:
```

Comment:
You may have to add this change to the SecureTransport `SSLContext` implementation as well.  We also need a `CHANGES.rst` entry saying that all `SSLContext.load_verify_locations()` implementations raise `urllib3.exceptions.SSLError` on a failure.